source/gui/__init__.py

```diff
@@ -804,7 +804,7 @@ class LauncherDialog(
 	helpId = "InstallingNVDA"
 
 	def __init__(self, parent):
-		super(LauncherDialog, self).__init__(parent, title=versionInfo.name)
+		super(LauncherDialog, self).__init__(parent, title=f"{versionInfo.name} {_('Launcher')}")

```

Comment:
possibly a controversial change? I added this to be more descriptive and so that we could test the "wait for speech" with something more unique